sync-tool/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "php-yaoi/php-yaoi": "1.0.18",
+        "chuyskywalker/rolling-curl": "^3.1"
```

Comment:
Ok, I just wanted to isolate sync-tool as end-user irrelevant utility. Will move it to main require-dev if you wish.